packages/kit/src/runtime/client/client.js

```diff
@@ -902,7 +902,7 @@ export function create_client(app, target) {
 	/**
 	 * @param {{
 	 *   url: URL;
-	 *   type: import('@sveltejs/kit').NavigationType;
+	 *   type: import('@sveltejs/kit').Navigation["type"];
```

Comment:
I'm not sure about the types here, looking at the codebase it seems that `NavigationType` would be correct?
https://github.com/sveltejs/kit/blob/5f96fe910007a0f35d75551daaf4e7d4f773d179/packages/kit/types/index.d.ts

But then you look a few lines down from where you changed and it uses the `Navigate` type :thinking:

https://github.com/sveltejs/kit/blob/9d8376f22d0d8db42327ce422ccb44d32cc91c04/packages/kit/src/runtime/client/client.js#L912

Reply:
Ideally it should be `BeforeNavigate["type"]` because that `navigation` object on L914 will be passed down to `before_navigate` callbacks. But both `Navigation` and `BeforeNavigate` have the same ts type for `type` property. Lmk if you think it should be updated to `BeforeNavigate["type"]`

Reply:
It's fine as-is 👍